docs/src/HealthTableGeneral.md

```diff
@@ -0,0 +1,30 @@
+# HealthTable: Tables.jl Interface (General)
+
+## The `HealthTable` Struct
+
+The core of the interface is the `HealthTable` struct. 
+
+```julia
+@kwdef struct HealthTable <: Tables.AbstractTable
+    source::DataFrame
+    omopcdm_version::String
+    function HealthTable(source)
+        # code goes here
+        return new(source, omopcdm_version)
+    end
+end
+```
+
+## `Tables.jl` API Implementation
```

Comment:
For each of these functions, could you also add the docstring information for the HealthTable implementation that you have drafted in the healthtable.jl file? Here's the documenter.jl syntax for this: https://documenter.juliadocs.org/stable/man/syntax/